### ISSUE TITLE
Kotlin library

### ISSUE THREAD

**Issue by dverdugo85** [koush/ion#911]
Hello @koush  are you have  a plan to convert the library to kotlin??    for the future of kotlin multiplatform Regards!!

**Comment by koush** [comment]
Yes, I've been playing with Kotlin quite lately a bit for this exact purpose. I'm not sure how I want to approach it, whether it's a rewrite or bindings.

**Comment by O330oei** [comment]
Kotlin library #916
